lib/src/components/TextArea/text-area.css.ts

```diff
@@ -0,0 +1,13 @@
+import { style } from "@vanilla-extract/css";
+
+export const textAreaStyle = style({
+    border: '1px solid #082D59',
+    borderRadius: '10px',
+    padding: '20px',
+    fontFamily: 'Proxima Nova',
+    fontStyle: 'normal',
+    fontWeight: '400',
+    fontSize: '14px',
+    lineHeight: '140%',
+    color: '#526C8B'
```

Comment:
let the global theme of consumer dictate the typography, if components start polluting typography the consumers will have the burden to override them.

Eg: If my project uses Roboto in body, I would expect textarea to use the same unless i explicitly specify the font via className.

Helps keep hover un-opnionated

Reply:
PS: I just realized that figma designs have those styles in place, but we do not want it to be exactly as in design, leaving some room for flexibility and extensibility helps